model-server/src/main/kotlin/org/modelix/model/server/handlers/ContentExplorer.kt

```diff
@@ -202,12 +204,13 @@ class ContentExplorer(private val client: IModelClient, private val repoManager:
                         th { +"Value" }
                     }
                 }
-                for (referenceRole in node.getReferenceRoles()) {
-                    tr {
-                        td { +referenceRole }
-                        td {
-                            // TODO MODELIX-387
```

Comment:
Is this ticket fixed after the PR? Not mentioned in the PR description.

Reply:
This PR was only possible, because that issue was fixed beforehand. It's part of this PR's issue description.